DependencyInjection/SerializerCompilerPass.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the FOSJsRoutingBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\JsRoutingBundle\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\Serializer\Encoder\JsonEncoder;
+use Symfony\Component\Serializer\Serializer;
+
+/**
+ * Class SerializerCompilerPass
+ *
+ * @author Miguel Angel Garzón <magarzon@gmail.com>
+ */
+class SerializerCompilerPass implements CompilerPassInterface
+{
+    const SERIALIZER_SERVICE_ID = 'fos_js_routing.serializer';
```

Comment:
you should not add such constant. There is no reason to make this public (and private constants are not available in the targeted PHP versions)